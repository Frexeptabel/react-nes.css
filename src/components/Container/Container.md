Container example:

```js
<Container>This is a container</Container>
<Container title="I'm a container">This is a container</Container>
<Container dark >This is a container</Container>
<Container rounded >This is a container</Container>
```
