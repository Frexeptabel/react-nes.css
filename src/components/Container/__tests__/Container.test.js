import React from "react";
import Container from "../index";
import renderer from "react-test-renderer";

test("The Container should be rendered properly", () => {
  const component = renderer.create(<Container>Test</Container>);
  expect(component).toMatchSnapshot();
});

test("The Container should react to its props", () => {
  let component = renderer.create(<Container dark>Test</Container>);
  expect(component).toMatchSnapshot();

  component = renderer.create(<Container rounded>Test</Container>);
  expect(component).toMatchSnapshot();

  component = renderer.create(<Container title="Hallo">Test</Container>);
  expect(component).toMatchSnapshot();
});
