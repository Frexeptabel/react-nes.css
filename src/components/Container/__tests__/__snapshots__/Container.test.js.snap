// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Container should be rendered properly 1`] = `
<div
  className="nes-container"
>
  Test
</div>
`;

exports[`The Container should react to its props 1`] = `
<div
  className="nes-container is-dark"
>
  Test
</div>
`;

exports[`The Container should react to its props 2`] = `
<div
  className="nes-container is-rounded"
>
  Test
</div>
`;

exports[`The Container should react to its props 3`] = `
<div
  className="nes-container with-title"
>
  <p
    className="title"
  >
    Hallo
  </p>
  Test
</div>
`;
